app/src/main/res/layout/activity_search_time.xml

```diff
@@ -22,4 +31,4 @@
         android:textColor="@color/light_grey"
         android:text="@string/pick_date"/>
 
-</LinearLayout>
\ No newline at end of file
+</LinearLayout>
```

Comment:
unneeded change! Rest LGTM!

Reply:
> unneeded change! Rest LGTM!

It was automatically done when I commit the file, Should I revery it if necessary?

Reply:
Let it be. CRLF changes are allowed